src/rcx/src/netRC.cpp

```diff
@@ -593,7 +590,6 @@ uint extMain::getCapNodeId(dbNet* net, dbBTerm* bterm, dbITerm* iterm,
     uint id = bterm->getId();
     uint capId = _btermTable->geti(id);
     if (capId > 0) {
-//(dbCapNode::getCapNode(_block, capId))->incrChildrenCnt();
 #ifdef DEBUG_NET_ID
```

Comment:
All this code is disabled by this ifdef.

Reply:
This ifdef's only disable writing to the file `fp` -- it ends two lines down.

The code I had to modify/remove is related to:
```
if (cap->getNet()->getId() == DEBUG_NET_ID)
```
which on itself is an odd check since two lines above there is:
```
if (cap->getNet()->getId() == _debug_net_id) {
```